packages/react-next/etc/react-next.api.md

```diff
@@ -1457,24 +1457,23 @@ export type IPositioningContainerTypes = IPositioningContainerProps;
 
 // @public (undocumented)
 export interface IRating {
+    rating: number;
 }
 
 // @public
-export interface IRatingProps extends React.AllHTMLAttributes<HTMLElement> {
+export interface IRatingProps extends React.HTMLAttributes<HTMLDivElement> {

```

Comment:
This seems a little arbitrary; can we add back the "all" html attributes here unless there's a good reason?

Reply:
"All" is misleading, at least for this component: unknown attributes are applied to the root div (not an input or something), and there are a lot of extra things in AllHTMLAttributes that aren't valid for divs. It was also preventing deprecation warnings from showing up for `min` since *some* HTML element also happens to have a `min` prop.

Reply:
hmm ok.